test/test_relevancy_api.py

```diff
@@ -0,0 +1,69 @@
+# coding: utf-8
+
+"""
+    TeePublic V3 search Api Documentation
+
+    No description provided (generated by Swagger Codegen https://github.com/swagger-api/swagger-codegen)  # noqa: E501
+
+    OpenAPI spec version: 1.0.9
+    
+    Generated by: https://github.com/swagger-api/swagger-codegen.git
+"""
+
+
+from __future__ import absolute_import
+
+import unittest
+
+import swagger_client
+from swagger_client.api.relevancy_api import RelevancyApi  # noqa: E501
+from swagger_client.rest import ApiException
+
+
+class TestRelevancyApi(unittest.TestCase):
+    """RelevancyApi unit test stubs"""
+
+    def setUp(self):
+        self.api = swagger_client.api.relevancy_api.RelevancyApi()  # noqa: E501
+
+    def tearDown(self):
+        pass
+
+    def test_delete_v1_relevancy_configuration(self):
```

Comment:
Is this codegen ? I think it is good if we can make use to this test to make sure the client code is ready to be used. We can setup circleCI in our next PR/upgrade.

Reply:
Yes, it's generated by codegen.